normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +31,46 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    def test_it_validates_arguments(self):
```

Comment:
@rehandalal ummm, are these on the right track?

I want to test the serializer with 3 different sets of data -- 
1. with an invalid action name, as it would not be able to find an arguments_schema to validate against
2. with a valid action, but invalid arguments, so it returns the right error formatting for `arguments`
3. with all valid data so it raises no errors

Is there a concept of isolating these different scenarios in pytest? Like in JavaScript you would wrap them all in a `describe` block then isolate each case into an `it` block.